cypress/integration/customTypes/00-create.spec.js

```diff
@@ -1,24 +1,43 @@
 describe("Custom Types specs", () => {
-  const name = "My Test";
-  const id = "my-test";
+  let name = "My Test";
+  let id = "my-test";
+
   beforeEach(() => {
+    const ID = Math.floor(10000 + Math.random() * 90000);
+    name = `My Test ${ID}`;
+    id = `my-test-${ID}`;
+
     cy.clearLocalStorageSnapshot();
     cy.cleanSliceMachineUserContext();
     cy.task("rmrf", `customtypes/${id}`);
```

Comment:
I'd recommend checking `rmrf` works, as last time I used it it required passing the path to the project too

Reply:
changed